components/blocks/how-to-tabs/how-to-tab/how-to-tab.hbs

```diff
@@ -0,0 +1,23 @@
+{{#each tabs as |tab|}}
+<div class="ht-tab{{#if variant}} {{variant}}{{/if}}">
+  {{#unless js}}<input type="checkbox" id="ht-tab-tr{{ tab.index }}" class="ht-tab-tr a11y--h" {{#if tab.open}}
+    checked{{/if}}>{{/unless}}
+  <{{#if js}}button{{else}}label for="ht-tab-tr{{ tab.index }}" {{/if}} class="ht-tab-h">
+    <div class="ht-tab-t">{{ tab.title }}</div>
+    <div class="ht-tab-ic">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path class="closed" d="M24 10h-10v-10h-4v10h-10v4h10v10h4v-10h10z"> </path>
+        <path class="open" d="M0 10h24v4h-24z"> </path>
+      </svg>
+      <div class="ht-tab-subinfo">
+        {{!-- <div class="closed">show</div>
+        <div class="open">hide</div> --}}
+        <span>show</span>
+      </div>
+    </div>
+  </{{#if js}}button{{else}}label{{/if}}>
+  <div class="ht-tab-c">
+    {{{ tab.content }}}
+  </div>
+</div>
+{{/each}}
```

Comment:
Check how you're ending these files, I see its yelling about 'no end of file' but its not doing it for others that end the same